.travis.yml

```diff
@@ -1,12 +1,8 @@
 sudo: false
+os: osx
+osx_image: xcode8.2
 language: node_js
 node_js:
   - "6"
-addons:
-  # This is the latest version of Firefox that currently supports Happo.
-  # https://github.com/SeleniumHQ/selenium/issues/2559
-  firefox: "47.0.1"
-before_script:
-  - 'sh -e /etc/init.d/xvfb start'
```

Comment:
Removing happo will kill visual diffs.

Reply:
what happens if we change the `travis ` script to `npm run --silent test && npm run --silent lint && npm run --silent happo` (and keep the firefox and xvfb)?

Reply:
react-sketchapp used to fail on ubuntu, which is why i switched it to osx. Xvfb is not available on osx https://github.com/travis-ci/travis-ci/issues/7277

Reply:
@mathieudutour what about the build status?
Visual diffs with happo have been disabled for a while. I assume because they are broken. React sketchapp does not build on linux. The happo script does not work on osx.
I think in the interest of the value provided by the successfulness of build status, react primitives should give up the idea of visual diffs for now. And bring it back in a more cross platform way.